Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -56,7 +56,9 @@ void doDepositionShapeN(const GetParticlePosition& GetPosition,
     const bool do_ionization = ion_lev;
     const amrex::Real dxi = 1.0/dx[0];
     const amrex::Real dzi = 1.0/dx[2];
+#if !(defined WARPX_DIM_RZ)
     const amrex::Real dts2dx = 0.5*dt*dxi;
+#endif
```

Comment:
Fixes the warning:
`./Source/Particles/Deposition/CurrentDeposition.H:59:23`: warning: unused variable ‘dts2dx’ [-Wunused-variable]
     const amrex::Real dts2dx = 0.5\*dt\*dxi;